app/templates/client/app/scripts/directives/loadingbartext.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+/**
+ * @ngdoc directive
+ * @name clientApp.directive:loadingBarText
+ * @description
+ * # loadingBarText
+ */
+angular.module('clientApp')
+  .directive('loadingBarText', function () {
+    return {
+      retrict: 'EA',
+      template: '<div class=\"splash-screen\" ng-show="isLoading">{{text}}</div>',
+      controller: function($scope) {
```

Comment:
- Isolate the construction and ui implementation into the directive. (Architecture based on Web components)
- We can easy extend the directive. 
- Inject the controller in the directive pattern will be a standard in Angular 2.0, already used in AngularDart